datafusion/core/src/execution/context.rs

```diff
@@ -341,6 +341,60 @@ impl SessionContext {
                     ))),
                 }
             }
+
+            LogicalPlan::SetVariable(SetVariable {
+                variable, value, ..
+            }) => {
+                let config_options = &self.state.write().config.config_options;
+
+                let old_value =
+                    config_options.read().get(&variable).ok_or_else(|| {
+                        DataFusionError::Execution(format!(
+                            "Unknown Variable {}",
```

Comment:
```suggestion
                            "Can not SET variable: Unknown Variable {}",
```

Reply:
didn't aware, that it's a commit suggestion lol